server/events/comment_parser.go

```diff
@@ -342,11 +368,13 @@ Examples:
   atlantis apply -d . -w staging
 
 Commands:
-  plan   Runs 'terraform plan' for the changes in this pull request.
-         To plan a specific project, use the -d, -w and -p flags.
-  apply  Runs 'terraform apply' on all unapplied plans from this pull request.
-         To only apply a specific plan, use the -d, -w and -p flags.
-  help   View help.
+  plan     Runs 'terraform plan' for the changes in this pull request.
+           To plan a specific project, use the -d, -w and -p flags.
+  apply    Runs 'terraform apply' on all unapplied plans from this pull request.
+           To only apply a specific plan, use the -d, -w and -p flags.
+  discard  Discards all plans in this PR as well as the atlantis locks.
```

Comment:
If you support -d, -w and -p then this should read.

```
Discards all plans in this PR and deletes all locks held by this PR.
To discard a specific project, use the -d, -w and -p flags.
```

Reply:
I was thinking, for simplicity, to only support general discard/unlock for the whole PR in this version. Would that be OK?

Reply:
Yes that sounds good.